android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -149,8 +157,8 @@ class CustomTabsToolbarFeature(
         @ColorInt navigationBarColor: Int?,
         @ColorInt readableColor: Int,
     ) {
-        if (updateToolbarBackground && toolbarColor != null) {
```

Comment:
I'm a little skeptical of removing `updateToolbarBackground` because it assumes this function is only called when it's supposed to be

Reply:
I believe this change would also prevent the `navigationBarColor` update below from being called in cases where it previously wouldn't be prevented

Reply:
> I believe this change would also prevent the navigationBarColor update below from being called in cases where it previously wouldn't be prevented

Good spot, I should have mentioned this in my original comment. We should only be updating the navigation bar theme if not in private mode, which originally contributed to the theming bug which is why this was changed